### DISCUSSION TITLE
Inserting image in header using closedxml

### DISCUSSION THREAD

**Discussion by ZubairImran** [ClosedXML/ClosedXML#2422] [Q&A]
Is it possible to insert the image in header. currently i could able to insert using worksheet addpicture function. But its creating drawing.xml structure. Can we insert image in header like word using closed xml?

**Comment by ZubairImran** [comment]
Hi Team,
When we add picture it is blocking the cell from further edit, is it possible to edit cell which is behind the image. Any pointer would be great help.

**Reply by ZubairImran** [reply]
@jahav @igitur , should we need to use VMLDrawing for this? Will this be added in ClosedXML near future? Is there already handling for VMLDrawingPart? i mean for adding the image in header.

**Comment by jahav** [comment]
It 's not currently possible and will not be added in near future.
